src/components/tagchip.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Chip from "@material-ui/core/Chip";
+import Link from "@material-ui/core/Link";
+import {Link as GatsbyLink} from "gatsby";
+
+//see gatsby-node.js
+const normalizeURL = (url) => encodeURIComponent(url.replace(/ /g, "-").toLowerCase());
+
+const TagChip = ({label, name, className}) => {
+  return (
+    <Link component={GatsbyLink} to={`/tag/${normalizeURL(name)}`} underline="none">
```

Comment:
Should this use each tag's color and bg from the configuration? I don't see those used anywhere else.

Reply:
I wanted to use those colors but it didn't look nice. I copied those colors from a gatsby blog using a light theme. I left them there for future reference if anyone finds a good way to use them.